README.md

```diff
@@ -22,90 +22,89 @@
+- [Integration with Vue](https://hyperformula.handsontable.com/guide/integration-with-vue.html#demo)
+- [Integration with Svelte](https://hyperformula.handsontable.com/guide/integration-with-svelte.html#demo)
 
 ## Installation and usage
 
-Install the library from [npm](https://www.npmjs.com/package/hyperformula):
+Install the library from [npm](https://www.npmjs.com/package/hyperformula) like so:
 
 ```bash
 npm install hyperformula
 ```
 
-Once installed, you can use it like this:
+Once installed, you can use it to develop applications tailored to your specific business needs. Here, we've used it to craft a form that calculates mortgage payments using the `PMT` formula.
 
 ```js
 import { HyperFormula } from 'hyperformula';
 
-// define the options
-const options = {
-  licenseKey: 'gpl-v3',
-};
+// Create a HyperFormula instance
+const hf = HyperFormula.buildEmpty({ licenseKey: 'gpl-v3' });
 
-// define the data
-const data = [['10', '20', '30', '=SUM(A1:C1)']];
+// Add an empty sheet
+const sheetName = hf.addSheet('Mortgage Calculator');
+const sheetId = hf.getSheetId(sheetName);
 
-// build an instance with defined options and data
-const hfInstance = HyperFormula.buildFromArray(data, options);
+// Enter the mortgage parameters
+hf.addNamedExpression('AnnualInterestRate', '8%');
+hf.addNamedExpression('NumberOfMonths', 360);
+hf.addNamedExpression('LoanAmount', 800000);
 
-// call getCellValue to get the calculation results
-const mySum = hfInstance.getCellValue({ col: 3, row: 0, sheet: 0 });
+// Use the PMT function to calculate the monthly payment
+hf.setCellContents({ sheet: sheetId, row: 0, col: 0 }, [['Monthly Payment', '=PMT(AnnualInterestRate/12, NumberOfMonths, -LoanAmount)']]);
 
-// print the result in the browser's console
-console.log(mySum);
+// Display the result
+console.log(`${hf.getCellValue({ sheet: sheetId, row: 0, col: 0 })}: ${hf.getCellValue({ sheet: sheetId, row: 0, col: 1 })}`);
 ```
 
-[Run this code in CodeSandbox](https://codesandbox.io/s/github/handsontable/hyperformula-demos/tree/develop/basic-usage)
-
-## What can it be used for?
-
-HyperFormula doesn't assume any existing user interface, making it a great general-purpose library that can be used in various business applications. Here are some examples:
-
-- Spreadsheets
-- Business logic builders
-- Forms and form builders
-- Computation notebooks
-- Smart documents
-- Educational apps
-- Online calculators
+[Run this code in CodeSandbox](https://codesandbox.io/p/sandbox/github/handsontable/hyperformula-demos/tree/feature/issue-1380/mortgage-calculator)
 
 ## Contributing
 
-Help us build the fastest and most flexible calculation engine for
-business web apps. Please read the [Contributing Guide](https://hyperformula.handsontable.com/guide/contributing.html) before making a pull request.
+Please read the [Contributing Guide](https://hyperformula.handsontable.com/guide/contributing.html) before making a pull request.
```

Comment:
Better: Contributions are welcome, but before you make them, please read the [Contributing Guide]() and accept the Contributor License Agreement.